ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
```

Comment:
Agree with @geoffthemedio here. A concise summary is sufficient for the "Key Changes" section, the details would add to much clutter here.